api/serverless.yml

```diff
@@ -0,0 +1,32 @@
+service: hemi-routing-api
+
+useDotenv: true
+
+custom:
+  esbuild:
+    packagePath: '../package.json'
+    watch:
+      pattern: ['./api/**/*.ts', '../lib/**/*.ts']
+  serverless-offline:
+    httpPort: 4000
+
+functions:
+  get-quote:
+    events:
+      - http: 
+          method: post
+          path: /v2/quote
```

Comment:
Don't we need CORS?

Reply:
It's actually setting `Access-Control-Allow-Origin` as `*` in the original handler. I guess once we know the url the interface will be deployed on we might set it to that.

Reply:
Additionally, I added `cors: true` on serverless config.